UnityProject/Assets/Scripts/Doors/DoorController.cs

```diff
@@ -95,8 +95,9 @@ public override void OnStartClient()
 			//                }
 			//            }
 
-			Awake();
-		}
+//            Awake();
+//            OnEnable();

```

Comment:
Is it really needed to comment these functions out?

Reply:
Actually it is, because OnEnable auto-registration is in now